core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java

```diff
@@ -218,9 +216,7 @@ public void onNext(HttpObject o) {
                 }
 
                 if (enableDateHeader && !newHeaders.contains(HttpHeaderNames.DATE)) {
-                    newHeaders.add(HttpHeaderNames.DATE,
-                                   DateTimeFormatter.RFC_1123_DATE_TIME.format(
-                                           ZonedDateTime.now(ZoneOffset.UTC)));
+                    newHeaders.add(HttpHeaderNames.DATE, HttpTimestampSupplier.currentTime());
```

Comment:
Thanks! /cc @jyblue

Reply:
Thank you @anuraaga 
This PR gave me a good education 👍